config/locales/en.yml

```
@@ -113,13 +113,8 @@ en:
         </p>
 
         <p class="govuk-body">
-          Make sure you are happy with the form before you make it live. After you
-          have made your form live:
+        After you have made your form live, completed forms will be sent to %{submission_email}.
```

Optionally you could indent this for readability
```suggestion
          After you have made your form live, completed forms will be sent to %{submission_email}.
```